README.md

```diff
@@ -155,6 +155,29 @@ https://github.com/panorama-ed/memo_wise. This project is intended to be a safe,
 welcoming space for collaboration, and contributors are expected to adhere to
 the [code of conduct](https://github.com/panorama-ed/memo_wise/blob/main/CODE_OF_CONDUCT.md).
 
+## Releasing
+
+To make a new release of `MemoWise` to Rubygems, first install the release
+dependencies:
+
+```shell
+bundle config set --local with 'release'
```

Comment:
I'm not sure I understand—why is this necessary?

Reply:
Ohh, this is so that `rake release` works?

Reply:
Yes -- I've added some clarification above, is this good now?